### PR TITLE
Fix the ci-framework collection

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Comment by cescgina** [comment]
recheck

File: tests/unit/module_utils/test_files/networking-definition-valid-all-tools-dual-stack.yml [hunk 1]
```diff
@@ -4,12 +4,6 @@ networks:
     network-v6: "fdc0:8b54:108a:c949::/64"
     gateway-v4: "192.168.122.1"
     gateway-v6: "fdc0:8b54:108a:c949::1"
-    dns-v4:
```
Comment (pablintino) [inline tests/unit/module_utils/test_files/networking-definition-valid-all-tools-dual-stack.yml:7]: Why is this file touched? Same goes for all the content in the unit tests folder. They will be moved for sure, but changed?

File: tests/unit/module_utils/test_files/networking-definition-valid-all-tools-ipv6-only.yml [hunk 2]
```diff
@@ -2,8 +2,6 @@ networks:
   network-1:
     network-v6: "fdc0:8b54:108a:c949::/64"
     gateway: "fdc0:8b54:108a:c949::1"
-    dns:
```
Comment (pablintino) [inline tests/unit/module_utils/test_files/networking-definition-valid-all-tools-ipv6-only.yml:5]: Same as previously.

File: tests/unit/module_utils/test_files/networking-definition-valid.yml [hunk 3]
```diff
@@ -18,9 +15,6 @@ networks:
   tenant:
     network: "172.19.0.0/24"
     gateway-v4: "172.19.0.1"
-    dns-v4:
```
Comment (pablintino) [inline tests/unit/module_utils/test_files/networking-definition-valid.yml:21]: Same as previously.

File: roles/ci_multus/molecule/default/converge.yml [hunk 4]
```diff
@@ -22,6 +22,7 @@
     cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
     cifmw_openshift_kubeconfig: "{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig"
     testpod_name: "pod-testnad"
+    cifmw_ci_multus_namespace: "ci-multus"
```
Comment (pablintino) [inline roles/ci_multus/molecule/default/converge.yml:25]: Is this needed? Why? Or better said, why moving into a collection needs this.

File: hooks/templates/service_config_ceph.yaml.j2 [hunk 5]
```diff
@@ -0,0 +1,34 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (pablintino) [inline hooks/templates/service_config_ceph.yaml.j2:1]: Seems to be a change not picked as a move, but this reminds me that maybe we should run the HCI job over this change before merging... @cjeanner @viroel @rebtoor

File: roles/ci_multus/molecule/default/converge.yml (hunk 4 above)
Comment (cescgina) [reply roles/ci_multus/molecule/default/converge.yml:25]: I needed it to run the molecule tests locally, but it was working fine in CI without it, I must have added it by mistake

File: tests/unit/module_utils/test_files/networking-definition-valid-all-tools-dual-stack.yml (hunk 1 above)
Comment (cescgina) [reply tests/unit/module_utils/test_files/networking-definition-valid-all-tools-dual-stack.yml:7]: I messed it up when resolving conflicts, let me put it back

File: hooks/templates/service_config_ceph.yaml.j2 (hunk 5 above)
Comment (cescgina) [reply hooks/templates/service_config_ceph.yaml.j2:1]: `podified-multinode-hci-deployment-crc` is already running in this PR, is this the job you mean?

Comment (pablintino) [reply hooks/templates/service_config_ceph.yaml.j2:1]: okok, I didn't spotted it when checking jobs. Yes, it's that one.

**Review by rebtoor — APPROVED** [review]

**Comment by pablintino** [comment]
Force merged cause the Zuul failure was probably because Zuul got out of resources. We tested the failed HCI job using a dedicated testproject commit resulting in a green pass: https://review.rdoproject.org/zuul/build/ba940c4f9d54467b99b8b523bf15905d

**Comment by SeanMooney** [comment]
note this will also affect infrawatch/stf jobs ill forward this to @elfiesmelfie to let her know

**Comment by cescgina** [comment]
> note this will also affect infrawatch/stf jobs ill forward this to @elfiesmelfie to let her know

thanks @SeanMooney if you have a link to that repo I can try to update it in preparation, for a while we'll keep the folder, so nothing should be broken

**Comment by SeanMooney** [comment]
ill also need to refactor my internal jobs but that fine. in future it would be good to reach out to the user of the ci framework so they can create the relevant patches before merging changes like this since we can test that with depends on

**Comment by SeanMooney** [comment]
https://github.com/infrawatch/service-telemetry-operator
emma is activly working on https://github.com/infrawatch/service-telemetry-operator/pull/531
to use extracted crc instead of nested.

**Pull request merged**